deployment-scenarios/cluster-internal-ssl/cluster-internal-ssl-configmap.yaml

```diff
@@ -0,0 +1,22 @@
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-internal-ssl-config
+data:
+  NEO4J_unsupported_dbms_ssl_system_ignore__dot__files: "true"
```

Comment:
does k8s do something funky with the volume mappings?  Or why is this needed?  I don't see any dotfiles in what you're mounting

Reply:
yes, yes it does, K8s creates the directories for the configmaps/secrets in its own locations and symlinks to them. It stores metadata in dot files which gets included.

The main reason for the wizardry is that they can update the symlink for a whole directory atomically at runtime if the ConfigMap or Secret is changed - obviously that's not relevant to Neo4j but there's no way to turn it off in K8s.